internal/jsre/deps/web3.js

```diff
@@ -3715,6 +3715,21 @@ var inputBlockNumberFormatter = function (blockNumber) {
     return utils.toHex(blockNumber);
 };
 
+var inputTraceFormatter = function (config) {
+    if (config !== undefined && Object.keys(config).length > 0) {
+        var validParams = collectTraceConfigFields().split(",");
```

Comment:
Exit early and un-indent: 
```js
if (config == undefined){ return undefined } 
if (object.Keys(config).length == 0) { return config } 
var validParams ... 
```

Reply:
I think the existing condition makes an early exit because the two inner conditions must be true to fall into the inner block.